label.go

```diff
@@ -0,0 +1,78 @@
+package nebula_go
+
+import (
+	"strings"
+)
+
+type Label struct {
+	Field   string `nebula:"Field"`
+	Type    string `nebula:"Type"`
+	Null    string `nebula:"Null"`
+	Default string `nebula:"Default"`
+	Comment string `nebula:"Comment"`
+}
+
```

Comment:
we can define `LabelName` here to represent the result of `show tags`/`show edges`.

Reply:
done